src/common/pool_map.c

```diff
@@ -2773,6 +2773,7 @@ pool_map_get_failed_cnt(struct pool_map *map, pool_comp_type_t type)
 static bool
 pool_map_empty(struct pool_map *map)
 {
+	D_ASSERT(map != NULL);
```

Comment:
Is this really a coverity warning?

Reply:
CID 316615 - Explicit null dereferenced https://scan8.coverity.com/reports.htm#v45872/p12225/fileInstanceId=40162812&defectInstanceId=11439672&mergedDefectId=316615

Reply:
Looks to already be fixed, and was in test code.

An assert is going to kill the server if it's hit, so if coverity is saying the code can reach this point with map being NULL then assert will simply change a signal 11 to signal 6 and won't make the code any more resilient.